content/rule_book/material_and_questions.md

```diff
@@ -19,215 +42,234 @@
 
-##### Chapter-Verse (CVR)
+The above does not mean that if the quizzer needs to provide a reference question that is a CVR, and their choice of interrogative word changes the question type to MACVR, that the quizzer has given a valid CVR. In this case, the validity changes based on the interrogative word because of how the required answer changes.
 
-- Must have a single answer.
-- The question must contain a word or phrase that exists in more than one verse in the same chapter.
-- Context is the single verse.
+##### Chapter Verse Reference (CVR)
 
-##### Multiple-Answer, Chapter-Verse (MACVR)
+- Must have a single answer
+- The question must contain a word or phrase that exists in more than 1 verse in the same chapter
+- Context is the single verse
 
-- Must meet all the requirements of a CVR except the single answer requirement.
-- Must also meet all the requirements of a MA except the unique word/phrase requirement.
+The question text should be in the form:
 
-##### Chapter (only) (CR)
+> According to [book name], chapter [chapter number], verse [verse number], [remainder of question text]
 
-The question must contain a word or phrase that exists more than once in the material and only once in the chapter.
+##### Chapter Reference (CR)
 
-##### Multiple-Answer, Chapter (only) (MACR)
+- Must have a single answer
+- Must contain a word or phrase that exists more than once in the material and only once in the chapter
 
-- Must meet all the requirements of a CR except the single answer requirement.
-- Must also meet all the requirements of a MA except the unique word/phrase requirement.
+The question text should be in the form:
+
+> According to [book name], chapter [chapter number], [remainder of question text]
+
+##### Multiple Answer Chapter Verse Reference (MACVR)
+
+- Must meet all the requirements of a CVR except the single answer requirement
+- Must also meet all the requirements of a MA except the unique word or phrase requirement
+
+##### Multiple Answer Chapter Reference (MACR)
+
+- Must meet all the requirements of a CR except the single answer requirement
+- Must also meet all the requirements of a MA except the unique word or phrase requirement
 
 #### Quote/Finish
 
-- Must be quoted word-perfect.
+Quote and Finish questions must be quoted word-perfect. Quizzers may go back and correct their answers if done in the question answer duration time period. The quizzer will be required to make 1 full, correct rotation; however, the quizzer can be called correct without 1 full, correct rotation if they have made a mistake only in 1 area of the verse and correct the mistake, making it abundantly clear to the quizmaster and answer judge that they know where their mistake was and what the corrected words are.
+
+Context is defined as the required verse, which is defined as the 1 or 2 verse range of the question, based on the question type.
+
+##### Quote (Q)
```

Comment:
QT right? or we are just redefining?

Reply:
"Q" because it's a "Quote" question. :)

Reply:
Fine, but no one labels them as such. This would be the first time.

Reply:
Go back far enough in time and it was "Q". Also, it's always been "Q" in CBQZ. :)